Index: app/src/main/res/values/colors.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?xml version=\"1.0\" encoding=\"utf-8\"?>\r\n<resources>\r\n    <color name=\"black\">#1A1B22</color>\r\n    <color name=\"white\">#FDFDFD</color>\r\n    <color name=\"blue\">#FDFDFD</color>\r\n    <color name=\"red\">#F56B6C</color>\r\n    <color name=\"gray\">#AEAFB4</color>\r\n    <color name=\"light_Gray\">#E6E8EB</color>\r\n</resources>\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/res/values/colors.xml b/app/src/main/res/values/colors.xml
--- a/app/src/main/res/values/colors.xml	(revision 6faf9bcd10b326b6046fe26ac6570aff3b33df89)
+++ b/app/src/main/res/values/colors.xml	(date 1772035582128)
@@ -2,7 +2,7 @@
 <resources>
     <color name="black">#1A1B22</color>
     <color name="white">#FDFDFD</color>
-    <color name="blue">#FDFDFD</color>
+    <color name="blue">#3772E7</color>
     <color name="red">#F56B6C</color>
     <color name="gray">#AEAFB4</color>
     <color name="light_Gray">#E6E8EB</color>
Index: .idea/.name
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/.name b/.idea/.name
deleted file mode 100644
--- a/.idea/.name	(revision 6faf9bcd10b326b6046fe26ac6570aff3b33df89)
+++ /dev/null	(revision 6faf9bcd10b326b6046fe26ac6570aff3b33df89)
@@ -1,1 +0,0 @@
-Practicum-Android-Diploma
\ No newline at end of file
Index: .idea/vcs.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/vcs.xml b/.idea/vcs.xml
deleted file mode 100644
--- a/.idea/vcs.xml	(revision 6faf9bcd10b326b6046fe26ac6570aff3b33df89)
+++ /dev/null	(revision 6faf9bcd10b326b6046fe26ac6570aff3b33df89)
@@ -1,6 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<project version="4">
-  <component name="VcsDirectoryMappings">
-    <mapping directory="" vcs="Git" />
-  </component>
-</project>
\ No newline at end of file
